module/VuFind/src/VuFind/Controller/AbstractSearch.php

```diff
@@ -346,7 +346,10 @@ protected function getSearchResultsView($setupCallback = null)
     {
         $view = $this->createViewModel();
         $config = $this->getConfig($this->getOptionsForClass()->getFacetsIni());
-        $view->multiFacetsSelection = (bool)($config->Results_Settings->multiFacetsSelection ?? false);
+        // Allows string value
+        $multiFacetsSelection = $config->Results_Settings->multiFacetsSelection ?? 'false';
+        $view->multiFacetsSelectionEnabled = $multiFacetsSelection !== 'false' && $multiFacetsSelection;
```

Comment:
Could we possibly avoid the need for two separate variables with: `($multiFacetsSelection !== 'false' && $multiFacetsSelection) ? $multiFacetsSelection : false`? Maybe it would even make sense to have a `getMultiSelectionValueFromConfig($config)` support method to contain this logic so we don't have to repeat it in two places.